docs/sdk/src/reference_docs/tokens_in_substrate.rs

```diff
@@ -0,0 +1,97 @@
+// This file is part of polkadot-sdk.
+//
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! # Tokens in Substrate
+//!
+//! This reference doc serves as a high-level overview of the token-related logic in Substrate and
```

Comment:
I think it would be quite cool to link this ref doc with this paragraph and make a full circle: https://github.com/paritytech/polkadot-sdk/blob/816c072abd99947e466e611cd96fa0db918f1854/docs/sdk/src/reference_docs/frame_pallet_coupling.rs#L143-L146

Reply:
For example, I would start with something like this: 

- To just launch a FRAME based runtime that is a cryptocurrency, all you have to do is to add `pallet_balances` to your runtime.
- To write custom pallets that interact with that currency, you can tightly couple them with balances
- To be more generalized, we use traits (and at this point you link `frame_pallet_coupling.rs`)